tests/basic/telemetry_test.go

```diff
@@ -233,94 +236,88 @@ var _ = Describe("{ProfileOnlyDiags}", func() {
 	var newDiags string
 	var err error
 	var pxInstalled bool
-	var skipTest = false
 
-	It("has to collect, validate profile diags on S3", func() {
+	It("has to collect, validate profile only diags on S3", func() {
 		contexts = make([]*scheduler.Context, 0)
+
 		// One node at a time, collect diags and verify in S3
 		for _, currNode := range node.GetWorkerNodes() {
 			pxInstalled, err = Inst().V.IsPxInstalled(currNode)
 			if err != nil {
-				logrus.Debugf("Could not get PX status on %s", currNode.Name)
+				logrus.Debugf("Could not get PX status on node [%s]", currNode.Name)
 			}
+
 			if pxInstalled {
-				// Get the most recent profile diags for comparison
-				Step(fmt.Sprintf("Check latest profile diags on node %v", currNode.Name), func() {
-					logrus.Infof(" Getting latest profile  diags on %v", currNode.Name)
-					existingDiags, err = telemetryRunCmd(fmt.Sprintf("ls -t /var/cores/*-*.{stack,heap}.gz | head -n 2"),
-						currNode, nil)
+				checkDiagsCmd := "ls -t /var/cores/*-*.{stack,heap} | head -n 2"
+				if isTelemetryEnabledInStorageCluster && TelemetryEnabled(currNode) {
+					checkDiagsCmd = fmt.Sprintf("ls -t /var/cores/*-*.{stack,heap}.gz | head -n 2")
```

Comment:
Can remove the fmt.Sprintf().. just use the string...

Reply:
Done